PowerAux/src/main.cpp

```diff
@@ -36,10 +41,10 @@ int main() {
             PRINT("main thread loop \r\n");
         #endif //TESTING
 
-        canParser.powerAuxExampleStruct.a++;
-        canParser.powerAuxExampleStruct.b += 2;
-        canParser.powerAuxExampleStruct.c += 3;
-        canParser.powerAuxExampleStruct.d += 4;
+        bms_can_parser.powerAuxExampleStruct.a++;
+        bms_can_parser.powerAuxExampleStruct.b += 2;
+        bms_can_parser.powerAuxExampleStruct.c += 3;
+        bms_can_parser.powerAuxExampleStruct.d += 4;
 
         thread_sleep_for(MAIN_LOOP_PERIOD_MS);
```

Comment:
Change this to use the `ThisThread::sleep_for()` method which we now have access to with RTOS enabled